static/js/services/select2-search.js

```diff
@@ -88,17 +89,44 @@ angular.module('inboxServices').factory('Select2Search',
           });
       };
 
-      var resolveInitialValue = function(selectEl) {
+      var resolveInitialValue = function(selectEl, initialValues) {
+        // TODO: add support for multiples values here
```

Comment:
Fix or raise issue. Or just remove if this feature isn't needed in the current product.